docs/en/wallet-instance-attestation.rst

```diff
@@ -5,163 +5,85 @@
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-In this section is described the format of the Wallet Instance Attestation and
-how it is issued by the Wallet Provider.
+
+This section describes the format of the Wallet Instance Attestation and how it is issued by the Wallet Provider.
 
 .. figure:: ../../images/dynamic_view_sequence_wallet_instance_attestation.svg
    :name: sequence diagram for Wallet Instance Attestation request
-   :alt: The figure shows the sequence diagram for issuing a Wallet Instance Attestation.
-         The steps will be described below.
+   :alt: The figure shows the sequence diagram for issuing a Wallet Instance Attestation. The steps will be described below.
    :target: https://www.plantuml.com/plantuml/uml/XPB1RzKm3CRl-IlCJY3nn7s7QOZ3118IGi0kkxYDLLcqJd2SLMz_FLvV6r7AnDN-_Fi-ExajXcfr6iEhh3XC24Rf2Kmh1QoMf4uTQGZPLTnpHZ6u-bv8hm0Br7tz7iUH33wAGwMdHJBpFpLVD3roN35p5qA5qusBhtsQZN7a9uBvekMLzo19GUbNfMBlib8X1_PAaUHveeIPJpTpTmrtPDjiNdrW8iE8Xc7kJgvoeyzh1VeaXYmimnyqi7EcyXP-qddnPAN9EruXYJcnsEhdf1yUrqbqC3MjnM3aOgxT5hmZ8NNrWix8MhQcH_zwMGyaIK-U5KwNgRNGB3yeFIF-kZYyBuNKE4a3VRh_5h0tVbpoTRiROLE__Y_eZOTP9W_RyZOpa5GM4YhbA2uy25fLQgrXkmDANDe7OClN7ktbXO-FyJ8jqluYpguDtVJSFc9y42MCPx04gJDa0Q5vz_LkIMATnjy0
 
-- **Message 1**: The User initializes the Wallet Instance.
-  In particular, this process happens after the Wallet Instance installation and after the expiration of the Wallet Instance Attestation
-  is launched and every time the User wants to request or present
-  (disclose) a credential.
-- **Message 2-3**: The Wallet Instance obtains metadata about its Wallet
-  Provider. Among these, we also find the list of supported algorithms,
-  public keys, endpoints.
-- **Message 4**: The Wallet Instance verifies that the Wallet Provider is
-  trustworthy by resolving the provider's trust chain up to the anchor
-- **Message 5-7**: The Wallet Instance instantiates a new key pair on its TEE
-  and requests a ``nonce`` from the Wallet Provider (as a measure
-  against replay attacks).
-- **Message 8**: The Wallet Instance generates a Wallet Instance Attestation
-  Request, in JWS format, signed with the private key associated with the
-  public key for which it wants to obtain the attestation containing
-  the ``nonce`` and all the required parameters.
-- **Message 9-13**: The Wallet Instance sends the Wallet Instance Attestation
-  Request to the Wallet Provider which verifies its validity and
-  issues the signed attestation.
-- **Message 13-14**:The Wallet Instance receives the Wallet Instance
-  Attestation signed by the Wallet Provider and proceeds with a formal
-  verification.
-- **Message 15**:The Wallet Instance Attestation is ready to be consumed.
+-
 
+ **Message 1**: The User initializes the Wallet Instance. In particular, this process happens after the Wallet Instance installation and after the expiration of the Wallet Instance Attestation is launched and every time the User wants to request or present (disclose) a credential.
```

Comment:
```suggestion
 **Message 1**: The User initializes the Wallet Instance. In particular, this process happens after the Wallet Instance installation and after the expiration of the Wallet Instance Attestation is launched and every time the User wants to request or present a credential.
```

Reply:
since the attributes in the credential are disclosed, while the entire credential is presented in its original status